api/patients/views.py

```diff
@@ -1002,6 +1002,27 @@ def load_control_param(request):
     return JsonResponse({"results": unique_month_result})
 
 
+def load_selected_control_params(request):
+    request_data = json.loads(request.body)
+    card_pk = request_data.get("card_pk") or None
+    data_params = CardControlParam.get_patient_control_param(card_pk)
+    control_param = PatientControlParam.get_contol_param_in_system()
+    for element in control_param:
+        if not data_params.get(element["id"], None):
+            data_params[element["id"]] = {"title": element["title"], "selected": False, "isGlobal": False}
+    result = [{"id": k, "title": v.get("title", ""), "isSelected": v.get("selected", False), "isGlobal": v.get("isGlobal", False)} for k, v in data_params.items()]
+    return JsonResponse({"results": result})
+
+
+def save_patient_control_params(request):
+    request_data = json.loads(request.body)
+    card_pk = request_data.get("card_pk") or None
+    selected_params = request_data.get("selectedParams") or None
+    CardControlParam.save_patient_control_param(card_pk, selected_params)
+
+    return JsonResponse({"results": ""})
```

Comment:
Ну хотя бы тогда уж так, а то результат совсем странный)
```suggestion
    return JsonResponse({"ok": True})
```